src/Microsoft.AspNetCore.Mvc.Razor/Internal/MvcRazorLoggerExtensions.cs

```diff
@@ -24,18 +33,59 @@ public static class MvcRazorLoggerExtensions
         private static readonly Action<ILogger, string, Exception> _tagHelperComponentInitialized;
         private static readonly Action<ILogger, string, Exception> _tagHelperComponentProcessed;
 
+
         static MvcRazorLoggerExtensions()
         {
-            _razorFileToCodeCompilationStart = LoggerMessage.Define<string>(
+            _viewCompilerStartCodeGeneration = LoggerMessage.Define<string>(
                 LogLevel.Debug,
                 1,
                 "Code generation for the Razor file at '{FilePath}' started.");
 
-            _razorFileToCodeCompilationEnd = LoggerMessage.Define<string, double>(
+            _viewCompilerEndCodeGeneration = LoggerMessage.Define<string, double>(
                 LogLevel.Debug,
                 2,
                 "Code generation for the Razor file at '{FilePath}' completed in {ElapsedMilliseconds}ms.");
 
+            _viewCompilerLocatedCompiledViews = LoggerMessage.Define<string[]>(
+                LogLevel.Debug,
+                3,
+                "Initializing Razor view compiler with the following compiled views: ('{ViewNames}').");
```

Comment:
Consider turning these into separate log statements rather than putting all of the views in a single message. It's a little janky to deal with things like this.